playground/src/routes/index.svelte

```diff
@@ -23,6 +23,13 @@
             name
             entity
             enjoys
+            image {
+              srcset
+              srcsetwebp
+              srcsetavif
+              placeholder
+              aspectratio
+            }
```

Comment:
We should probably think about how to provide fragments in the future so people don't have to manually add these types for things like svimg could maybe just do
```
image {
 ...svimgFragment
}
```

Reply:
Definitely - that's a cool idea